hack/olm-registry/olm-artifacts-hypershift-template.yaml

```diff
@@ -145,6 +145,8 @@ objects:
           mediatype: ''
         publisher: Red Hat
         sourceType: grpc
+        grpcPodConfig:
+          securityContextConfig: restricted 
```

Comment:
There's already a `grpcPodConfig` for AVO's catalogsources, so we'd want to add `securityContextConfig` to ~line 123 for this file, but similar change for the others

/hold

Reply:
thanks, updated.